packages/geo/src/lib/layer/utils/image-watcher.ts

```diff
@@ -60,4 +68,23 @@ export class ImageWatcher extends Watcher {
       }
     }
   }
+
+  private handleLoadError(event) {
+
+    if (!event.image.__watchers__) {
+      return;
+    }
+    const title = this.languageService.translate.instant(
+      'igo.geo.dataSource.unavailableTitle'
+    );
+    const message = this.languageService.translate.instant(
+      'igo.geo.dataSource.olLoadProblem', {value: event.target.params_.LAYERS}
```

Comment:
@josee666 why not use the same translation key.... unavailable ?

Reply:
We can, it will be ok, but i that case we know that is not 'unavailable' it's a loadind problem. I think it's more precise. But it's strange that the same problem, bad layer name, with optionFromCapabilities:true and optionFromCapabilities: false doesn't have the same error message. No problem to use the unavailable message, you can change it if you want.

Reply:
@PhilippeLafreniere18 what do you prefer?

Reply:
I think we can keep the same translation key "unavailable" (not sure if it really helps the user to know that it is unavailable because it's a loading error vs other kind of display errors)

Reply:
ok i change it! Thanks